AUTHORS.md

```diff
@@ -1,4 +1,4 @@
-This is a (possibly incomplete) list of all the contributors to python-dateutil,
+﻿This is a (possibly incomplete) list of all the contributors to python-dateutil,
```

Comment:
There are a bunch of weird unprintable characters at the beginning of this line. Is this something your editor is doing?

```python
with open('AUTHORS.md', 'rb') as f:
    print(f.read(60))
# Result:
# b'\xef\xbb\xbfThis is a (possibly incomplete) list of all the contribut'
```

Since the sequence is `0xEFBBBF`, I'm guessing that it's a Windows [Byte Order Mark](https://en.wikipedia.org/wiki/Byte_order_mark#UTF-8) indicating UTF-8. I recommend that you fix your editor so that it doesn't do this anymore.